_includes/navigation.html

```diff
@@ -37,6 +37,11 @@
                     <span class="label label-pill label-info">{{ site.data.future-episodes.size }}</span>
                 </a>
             </li>
+             <li class="nav-item">
+                <a class="nav-link" href="http://werocktech.com/awesome-tech-communities/" target="_blank">
+                    <i class="fa fa-handshake-o" aria-hidden="true"></i> Communities
```

Comment:
That icon is only available from font awesome `v4.7`

<img width="895" alt="screenshot 2017-06-13 13 06 05" src="https://user-images.githubusercontent.com/624760/27081616-25e92bbc-5039-11e7-859c-8c8a8e8925e0.png">

I will update **FontAwesome** as using `v4.5.0`